cmd/state/errors.go

```diff
@@ -95,7 +95,9 @@ func unwrapError(err error) (int, error) {
 	// unwrap exit code before we remove un-localized wrapped errors from err variable
 	code := errs.UnwrapExitCode(err)
 
-	if !locale.HasError(err) && isErrs && !hasMarshaller {
+	// Account for low level erros returned from 3rd party libraries that are not localized
+	unwrappedErrs := locale.UnwrapError(err)
+	if !locale.HasError(err) && isErrs && !hasMarshaller && len(unwrappedErrs) != 0 {
```

Comment:
This makes no sense. The first condition says `err` is has no localization in its chain, but the last condition is saying it needs to have at least one localization in its chain. I'm not sure what problem this is trying to solve, but please revisit.

Reply:
The underlying issue comes from adding tips to all errors. If the error is from cobra, adding a tip changes the error type and `isErrs` is now true.

I've added a fix that doesn't add tips to errors when there is no child command found since that will happen in this case, however other errors returned form cobra will have this problem.

We could add a check that the error does not implement our `errs.Error` interface before adding tips, but I was not sure if that would be appropriate.